packages/admin/src/pages/scheme/[schemeId]/manage-editors/index.page.tsx

```diff
@@ -0,0 +1,99 @@
+import InferProps from '../../../../types/InferProps';
+import { SummaryList } from 'gap-web-ui';
+import Meta from '../../../../components/layout/Meta';
+import CustomLink from '../../../../components/custom-link/CustomLink';
+import {
+  Action,
+  UnformattedEditorRow,
+  getServerSideProps,
+} from '../editors.getServerSideProps';
+import { Row } from 'gap-web-ui/dist/cjs/components/summary-list/SummaryList';
+
+export { getServerSideProps };
+
+const OWNER = 'Owner';
+
+const LinkToAction = ({ action }: { action: Action }) => (
+  <CustomLink
+    href={action.href}
+    dataCy={`cy_summaryListLink_${action.ariaLabel}`}
+    ariaLabel={action.ariaLabel ?? action.label}
+  >
+    {action.label}
+  </CustomLink>
+);
+
+const formatEditorRow = (acc: Row[], row: UnformattedEditorRow) => {
+  const mapRequired = typeof row.action !== 'string' && row.action;
+  let action = (
+    mapRequired ? <LinkToAction action={row.action as Action} /> : row.action
+  ) as Exclude<Row['action'], undefined>;
+
+  if (row.value === OWNER) action = '-';
+
+  return [...acc, { ...row, action }];
+};
+
+const ManageEditors = ({
+  editorRows,
+  schemeName,
+  schemeId,
+}: InferProps<typeof getServerSideProps>) => {
+  const mappedTableRows = editorRows.reduce(formatEditorRow, []);
```

Comment:
Correct me if I'm wrong but I think this evaluates to the same in all instances
```suggestion
const formatEditorRow = (row: UnformattedEditorRow) => {
  const mapRequired = typeof row.action !== 'string' && row.action;
  if (row.value === OWNER) return { ...row, action: '-' };
  if (mapRequired)
    return { ...row, action: <LinkToAction action={row.action as Action} /> };
  return row;
};

const ManageEditors = ({
  editorRows,
  schemeName,
  schemeId,
}: InferProps<typeof getServerSideProps>) => {
  const mappedTableRows = editorRows.map(formatEditorRow);
```